anesthetic/gui/plot.py

```diff
@@ -53,7 +53,7 @@ def update(self, logX, LX, i):
     def reset_range(self):
         """Reset the ranges of the higson plot."""
         xdata = self.curve.get_xdata()
-        self.ax.set_xlim(max(xdata), min(xdata))
+        self.ax.set_xlim(max(xdata), min(xdata[xdata != -numpy.inf]))
```

Comment:
Both of these are really good points. I'll try to remember the ``isfinite`` for future, as it's much neater (it might be worth  hunting through the code for rogue ``numpy.inf``s that could be fixed by this -- #70 might be a good place for such a change). I've removed ``min(...)`` from the rest of the codebase as far as appropriate.